test/functions/fcf/bug-23701.chpl

```diff
@@ -0,0 +1,10 @@
+record r {
```

Comment:
Feel free to ignore this, but I'd rename the test to reflect what it's checking and leave a comment inside it with the bug number (or at least a comment about what the test is for).  The former will make it easier to know this is a unit test and what it's related to if it fails in the future, while the latter will help ensure we don't get overzealous updating for language changes in the future and accidentally lose the purpose of the test.